tests/feature/no_sctrict_gherkin.feature

```diff
@@ -0,0 +1,9 @@
+Feature: No strict Gherkin Background support
+
+    Background:
+        When foo has a value "bar"
+        And foo is not boolean
+        And foo has not a value "baz"
+        Then foo has length equal 1
```

Comment:
also including even THEN in the background blows my mind - can't imagine the real use-case and why should we even test for such a silly thing?